catalogue/mpi/pdq/user-guide/quickstart.md

```diff
@@ -7,19 +7,19 @@ To follow along with this quickstart you'll need:
 
 There are a number of different combinations of fields that can be combined to search the MPI. A sample of these will be detailed with examples below. 
 
-## HTTP Header values
+### HTTP Header values
 
 To POST any search requests successfully you will need to set the following API Header values. There would be entered under the [Headers tab](https://learning.postman.com/docs/sending-requests/requests/#configuring-request-headers) in Postman or as a [custom HTTP header](https://www.soapui.org/docs/functional-testing/teststep-reference/http-request/headers/) in SOAPUI.
 
-| Field                     |  Value
-|---------------------------|:-------------------------------------------------------------|
-| Ocp-Apim-Subscription-Key |  \<your subscription key\>*                                    |
-| SOAPAction                |  http://apps.wales.nhs.uk/mpi/InvokePatientDemographicsQuery |
-| Content-Type              |  text/xml                                                    |
+ Field                     |  Value
+:--------------------------|:-------------------------------------------------------------
+ Ocp-Apim-Subscription-Key |  &ltyour subscription key\>                                
+ SOAPAction                |  http://apps.wales.nhs.uk/mpi/InvokePatientDemographicsQuery 
+ Content-Type              |  text/xml                                                    
 
-*\<your subscription key\>* is unique to you and is allocated when you sign up to the [NHS Wales Developer Portal](https://developer.nhs.wales/) and subscribe to sandbox api's under your profile (visible once you have signed in). If you use one of the tryit links on the portal for a sandbox api you will see that it is automatically included in each api call.
+<your subscription key>* is unique to you and is allocated when you sign up to the [NHS Wales Developer Portal](https://developer.nhs.wales/) and subscribe to sandbox api's under your profile (visible once you have signed in). If you use one of the tryit links on the portal for a sandbox api you will see that it is automatically included in each api call.
```

Comment:
```suggestion
{your subscription key}* is unique to you and is allocated when you sign up to the [NHS Wales Developer Portal](https://developer.nhs.wales/) and subscribe to sandbox api's under your profile (visible once you have signed in). If you use one of the tryit links on the portal for a sandbox api you will see that it is automatically included in each api call.
```
The opening angle bracket isn't escaped so the test won't display.
I suggest just switching it to curly braces as they're less problematic